chart/flux/values.yaml

```diff
@@ -202,8 +202,8 @@ kube:
     kind: Config
     preferences: {}
     users: []
-#for https://github.com/justinbarrick/fluxcloud/
-#additionalArgs:
+# for https://github.com/justinbarrick/fluxcloud/
```

Comment:
```suggestion
# For https://github.com/justinbarrick/fluxcloud/
```

To be in line with all other comments in this file. Makes everything a tiny bit prettier.